care/facility/api/viewsets/ambulance.py

```diff
@@ -1,6 +1,7 @@
 from django_filters import rest_framework as filters
 from rest_framework import serializers, status
 from rest_framework.decorators import action
+from rest_framework.generics import get_object_or_404
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.generics'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5398997)